### ISSUE TITLE
import statement formatting problems:

### ISSUE THREAD

**Issue by aleksey-bykov** [Microsoft/TypeScript#5947]
```
// given
import {
    numbers as bn,
    optional as bo,
    base as bb,
    array as ba,
    list as bl
} from '@bykov/basics';
```

```
// actual
import {
numbers as bn,
optional as bo,
base as bb,
array as ba,
list as bl
} from '@bykov/basics';
```

```
// expected
import {
    numbers as bn,
    optional as bo,
    base as bb,
    array as ba,
    list as bl
} from '@bykov/basics';
```

**Comment by SaschaNaz** [comment]
#4516 is also related.

**Comment by DanCorder** [comment]
Actually this may already be fixed by #4609

**Comment by mhegazy** [comment]
thanks @DanCorder!
